### ISSUE TITLE
Make Calendar WCAG 2.1 compatible

### ISSUE THREAD

**Issue by ExtAnimal** [bryntum/support#9998]
Standard: https://www.w3.org/TR/WCAG21/

More practical info: https://help.time.ly/event-management/how-to-set-up-calendar-design-for-accessibility-compliance/

 - [x] Add accessibility theme
 - [x] Make day header cells visitable as `role="columnheader"`
 - [x] Make day header cell's dayName visitable as `role="link"` as click links to DayView for that date
 - [x] Make day cells visitable as `role="gridell"`
 - [x] Make MonthView's week number visitable as `role="link"` as it links to the WeekView
 - [x] Make YearView month and week number buttons `role="link"` as they link to month and week view.
 - [x] Make YearView cells visible as `role="button"`

**Comment by rgallagherab** [comment]
Hi, I was checking back on an [issue filled by my colleague in March](url) about A11Y violations a large customer reported that need to be resolved, and I came across this recent issue.

I know that "Calendar" is a separate product from Scheduler Pro, but will this compliance effort also be applied to that tool?

**Comment by matsbryntse** [comment]
@rgallagherab The link you posted seems broken, can you please post it again and we'll take a look!
